cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -94,7 +97,8 @@ class PredicateFunctionsTest extends DocumentingTest {
       }
     }
     section("none()", "functions-none") {
-      p("`none()` returns true if the predicate holds for no element in the given list.")
+      p("`none()` returns true if the predicate holds for no element in the given list." +
+        " Returns `null` if the list is `null` or all of its elements are `null`.")
```

Comment:
"Returns `null` if the..." -> "`null` is returned if the ..."  (to make it a sentence, rather than a clause)